neqo-client/src/main.rs

```diff
@@ -492,13 +499,37 @@ fn main() -> Res<()> {
                 &urls,
             )?;
         } else if !args.download_in_series {
+            let mut token: Option<Vec<u8>> = None;
+
+            if resumption_test {
```

Comment:
I think that you want to do this differently now that I think on it more.

Rather than take a single token and reuse that on every subsequent request, you should instead use the multiconnect option when resumption is requested.  That keeps the logic here simpler:

```rust
let mut resumption_token = None;
for url in urls {
  token = old::old_client(..., token);
}
```

Reply:
> Rather than take a single token and reuse that on every subsequent request, you should instead use the multiconnect option when resumption is requested. That keeps the logic here simpler:

That's not how it works now. First URL is fetched to get the token, but all subsequent URL are fetched over a single connection.
 
> ```rust
> let mut resumption_token = None;
> for url in urls {
>   token = old::old_client(..., token);
> }
> ```

This wouldn't work for zerortt test because it downloads 40 files but expects exactly 2 handshakes. With this code we would have 40 handshakes.

Reply:
I see.  Oh well, carry on.